sdk/src/main/java/io/opentelemetry/sdk/trace/spi/TraceProviderSdk.java

```diff
@@ -17,13 +17,12 @@
 package io.opentelemetry.sdk.trace.spi;
 
 import io.opentelemetry.sdk.trace.TracerSdkProvider;
-import io.opentelemetry.trace.TracerProvider;
 import io.opentelemetry.trace.spi.TraceProvider;
 
 /** SDK implementation of the TracerProviderFactory for SPI. */
 public final class TraceProviderSdk implements TraceProvider {
   @Override
-  public TracerProvider create() {
+  public TracerSdkProvider create() {
```

Comment:
I don't think this is the expected returned type.

Reply:
Can you be more specific why? Instead of returning the interface it returns the concrete type (covariant return type). This is a common practice.

Reply:
@pavolloffay Please move this in a different PR and we can discuss there. We should recommend users to call with `OpenTelemetrySdk.getTracerProvider`

Reply:
I will revert it and cast the object in the builder instead. Are there any other comments for this PR?